web/themes/custom/cloudy/cloudy.info.yml

```diff
@@ -13,7 +13,7 @@ stylesheets-remove:
   - '@extlink/extlink.css'
 
 ckeditor_stylesheets:
-  - dist/style.bundle.css
+  - dist/cloudy.css
   - https://use.fontawesome.com/releases/v5.2.0/css/all.css
```

Comment:
I approved this, then thought... shouldn't we switch this to the locally compiled fontawesome files?